db/migrate/20190104162627_add_abp_to_impressions.rb

```diff
@@ -0,0 +1,6 @@
+class AddAbpToImpressions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :impressions, :abp, :boolean, default: false
+    add_index :impressions, :abp
+  end
+end
```

Comment:
`abp` is somewhat abstract. I'd prefer we give this a more semantically meaningful name.